internal/core/config/loader_test.go

```diff
@@ -448,7 +445,7 @@ var _ = Describe("Config Loader", func() {
 			signalFxAccessToken: abcd
 			monitors:
 			- type: collectd/mysql
-			  password: {"#from": "%s/agent/mysql-password.yaml", default: "s3cr3t"}
```

Comment:
How does this work on Windows?  I thought paths had to use backslashes?

Reply:
will obviously need to test to be 100% sure, which I'll get to in the future, but for now the test passes.